tests/inspect.rs

```diff
@@ -0,0 +1,31 @@
+use std::process::Command;
+use std::io::Write;
+use std::env;
+use std::path::Path;
+
+fn nm(path: &Path) -> Command {
+    let mut nm = Command::new("nm");
+    nm.arg("-g").arg("--defined-only").arg(path);
+    return nm
+}
+
+#[test]
+fn inspect() {
```

Comment:
I'm not sure whether it'd still be worth keeping this around, but curious for your thoughts!